tox.ini

```diff
@@ -0,0 +1,81 @@
+[tox]
```

Comment:
General on `tox.ini` - ideally, all actual options for testing/style are kept in `setup.cfg` and this does nothing more than define the tox environments that run them automatically. Also, comments on what things do would be most appreciated! Much of it is not obvious...

Reply:
I will try and add more comments to the tox.ini file.

Reply:
I've now added a lot more descriptive comments in the tox file